mmv1/products/certificatemanager/IssuanceConfigs.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'IssuanceConfigs'
+base_url: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs'
+create_url: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs?certificateIssuanceConfigId={{name}}'
+self_link: 'projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs/{{name}}'
+immutable: true
+description: |
+  Certificate represents a HTTP-reachable backend for a Certificate.
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+docs: !ruby/object:Provider::Terraform::Docs
+autogen_async: true
+import_format: ["projects/{{project}}/locations/{{location}}/certificateIssuanceConfigs/{{name}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'certificate_manager_issuance_configs'
+    primary_resource_id: 'default'
+    vars:
+      ca_name: 'my-ca'
+      pool_name: 'my-ca-pool'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
```

Comment:
Given the implementation of the resource, this is just the last section of that format & not the entire string. The `id` meta-field will hold the value with this pattern.

Reply:
Nice catch! Probably, It was misinterpreted in the API documentation. Thanks for pointing that out!